GoogleForCreators/web-stories-wp
# Media: Remove manual cache busting

## Context

> We've added manual cache busting in the past because we had to support older WordPress versions.

> Now that we are bumping the WP version requirement to WP 5.7+ we should be able to finally remove this manual cache busting.

See: https://github.com/GoogleForCreators/web-stories-wp/issues/4312#issuecomment-1160536112



## Summary

Removes cache bust parameter where it has been applied 



## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1.  Upload via local media pane "upload button". Close the modal. The pane should have the new media displayed (#4310)

2. #4109
- create new story
- select "Upload" from media panel
- browse for and upload an image
- insert this new image into page 
- ensure the new media shows under the "all" category and the "images" category for the media panel (no page refresh)


## Reviews

### Does this PR have a security-related impact?

No



### Does this PR change what data or activity we track or use?

No



### Does this PR have a legal-related impact?

No



## Checklist




---



Fixes #4312
